.github/workflows/bench.yml

```diff
@@ -17,7 +17,6 @@ on:
 jobs:
   pre_job:
     runs-on: ubuntu-latest
-    if: contains(github.event.pull_request.labels.*.name, 'performance')
```

Comment:
Needs to be readded, right?

Reply:
I move it down to the run session, so if performance label is provided, we build and run the bench, if not, we only build the bench.